src/Uno.UI/UI/Xaml/UIElement.Pointers.cs

```diff
@@ -768,6 +769,19 @@ internal void RaiseDrop(global::Windows.UI.Xaml.DragEventArgs args)
 		partial void PrepareManagedPointerEventBubbling(RoutedEvent routedEvent, ref RoutedEventArgs args, ref BubblingMode bubblingMode)
 		{
 			var ptArgs = (PointerRoutedEventArgs)args;
+
+#if __ANDROID__
+			if (this is not RootVisual &&
+				this.Parent is not UIElement &&
+				this.Parent is Android.Views.View viewParent &&
+				ptArgs.MotionEvent is { } motionEvent
+				)
+			{
+				// bubblingMode = BubblingMode.NoBubbling;
+				viewParent.OnTouchEvent(motionEvent);
```

Comment:
I don't  remember if it's required or not, but you might have to adjust coordinates on `motionEvent` before in order to make coordinates relative to the `viewParent` element. (You can follow what's done in native : clone it using the right [`obtain`](https://developer.android.com/reference/android/view/MotionEvent#obtain(android.view.MotionEvent)), then [`offsetLocation`](https://developer.android.com/reference/android/view/MotionEvent#offsetLocation(float,%20float)) and finally [`recycle`](https://developer.android.com/reference/android/view/MotionEvent#recycle())) it.